docs/root/start/start.rst

```diff
@@ -0,0 +1,154 @@
+disk. Then, build and run the Dockerfile, and test out Envoy by sending a
+request to port 10000::
+
+  $ docker build -t envoy-google-test:v1 .
+  $ docker run -d -p 10000:10000 envoy-google-test:v1
+  $ curl -v localhost:10000
+
+The Dockerfile will build a container containing the latest version of Envoy,
+and copy a basic Envoy configuration into the container. This basic
+configuration tells Envoy to route incoming requests to \*.google.com.
+
+
+Simple Configuration
+--------------------
+
+Envoy can be configured using a single YAML file passed in as an argument on the command line.
+
+The :ref:`admin message <envoy_api_msg_config.bootstrap.v2.Admin>` is required to configure
+the administration server. The `address` key specifies the :ref:`address <envoy_api_file_envoy/api/v2/core/address.proto>`
+which in this case is simply `localhost:9901`.
+
+.. code-block:: yaml
+
+  admin:
+    access_log_path: /tmp/admin_access.log
+    address:
+      socket_address: { address: 0.0.0.0, port_value: 9901 }
+
+The :ref:`static_resources <envoy_api_field_config.bootstrap.v2.Bootstrap.static_resources>` contains everything that is configured staticaly when Envoy starts,
+as opposed to the means of configuring resources dynamically when Envoy is running.
+
+.. code-block:: yaml
+
+    static_resources:
+
+The :ref:`listeners <envoy_api_file_envoy/api/v2/listener/listener.proto>` blah.
+
+.. code-block:: yaml
+
+      listeners:
+      - name: listener_0
+        address:
+          socket_address: { address: 0.0.0.0, port_value: 10000 }
+        filter_chains:
+        - filters:
+          - name: envoy.http_connection_manager
+            config:
+              stat_prefix: ingress_http
+              codec_type: AUTO
+              route_config:
+                name: local_route
+                virtual_hosts:
+                - name: local_service
+                  domains: ["*"]
+                  routes:
+                  - match: { prefix: "/" }
+                    route: { host_rewrite: www.google.com, cluster: service_google }
+              http_filters:
+              - name: envoy.router
+
+The :ref:`clusters <envoy_api_file_envoy/api/v2/cds.proto>` blah.
+
+.. code-block:: yaml
+
+      clusters:
+      - name: service_google
+        connect_timeout: 0.25s
+        type: LOGICAL_DNS
+        # Comment out the following line to test on v6 networks
+        dns_lookup_family: V4_ONLY
+        lb_policy: ROUND_ROBIN
+        hosts: [{ socket_address: { address: google.com, port_value: 443 }}]
+        tls_context: { sni: www.google.com }
+
+
+Using the Envoy Docker Image
+----------------------------
+
+Create a simple Dockerfile to execute Envoy, which assumes that envoy.yaml (described above) is in your local directory.
+
+.. code-block:: none
+
+  FROM envoyproxy/envoy:latest
+  RUN apt-get update
+  COPY envoy.yaml /etc/envoy.yaml
+  CMD /usr/local/bin/envoy -c /etc/envoy.yaml
```

Comment:
Add a link to the command line documents.

Reply:
Done